src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -473,7 +474,10 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
       isEditable: false,
       isSaving: false,
       isCollapsed: false,
+      isChanged: false,
     };
+    this.loadResponseStatusForQuestion(model);
```

Comment:
If we just want to fix the bug for not popping up the alert model when saving the second time, I think the root cause is the response status is not updated when saving. Therefore, it is not necessary to add `isChanged` to the model?

Reply:
In addition, I don't think this is the correct place to call the method.

- `get***` should not trigger any side-effect.
- `loadResponseStatusForQuestion` will be called twice I believe when the page is loaded.

Reply:
> If we just want to fix the bug for not popping up the alert model when saving the second time, I think the root cause is the response status is not updated when saving. Therefore, it is not necessary to add `isChanged` to the model?

I added the `isChanged` as an initial value for the model, since it is definitely unchanged upon creating the edit form model. Not related to the bug.

> In addition, I don't think this is the correct place to call the method.
> 
>     * `get***` should not trigger any side-effect.
> 
>     * `loadResponseStatusForQuestion` will be called twice I believe when the page is loaded.

Yep. I moved it to within `saveExistingQuestionHandler`.

Reply:
> I added the isChanged as an initial value for the model, since it is definitely unchanged upon creating the edit form model. Not related to the bug.

So the `isChange` field is added for better user experience?

Reply:
Yes